app/src/main/java/org/simple/clinic/patient/sync/PatientSyncApi.kt

```diff
@@ -24,6 +26,7 @@ interface PatientSyncApi {
       @Query("process_token") lastPullTimestamp: String? = null
   ): Call<PatientPullResponse>
 
+  @Timeout(value = 4000, unit = TimeUnit.MILLISECONDS)
```

Comment:
I prefer to keep it in milliseconds and also with timeout throughout the project mostly, we've been using milliseconds.

Reply:
We pass `TimeUnit.MILLISECONDS` only if we are passing a millisecond value. Since we are setting this value, setting it to `TimeUnit.SECONDS` is much more readable. Also, this timeout should be 5 seconds if I remember correctly, so do verify this and update accordingly.

Reply:
Also, I don't see a reason for it to be a millisecond value, since it's not a fraction. But it's no big deal, just confirm the timeout value and that should be good.